client/components/edit/datastream/DatastreamModal.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { Dialog, DialogTitle, Grid, IconButton } from "@mui/material";
+import CloseIcon from "@mui/icons-material/Close";
+import { useEditorContext } from "../../../context/EditorContext";
+import DatastreamUploadModalContent from "./DatastreamUploadModalContent";
+
+const DatastreamModal = () => {
```

Comment:
Is this a DatastreamUploadModal rather than just a DatastreamModal, or do you plan to embed all datastream functionality into this one component in future?

Reply:
I'm planning provide a mapping of this later. It's in my next pr.